boot/makebootable_test.go

```diff
@@ -728,10 +755,20 @@ version: 5.0
 	defer restore()
 
 	// set mock key sealing
-	sealKeyCalls := 0
-	restore = boot.MockSecbootSealKey(func(key secboot.EncryptionKey, params *secboot.SealKeyParams) error {
-		sealKeyCalls++
-		c.Check(key, DeepEquals, myKey)
+	sealKeysCalls := 0
```

Comment:
This var is not actually used in the test. The test also tests the error case and will error after the first seal call so the switch below with "case 2" is never hit. I guess this should just be split into erroring once for "sealKeysCall" calls 1 and then calls 2.